go/vt/vtorc/inst/analysis.go

```diff
@@ -90,17 +91,19 @@ const (
 
 // ReplicationAnalysis notes analysis on replication chain status, per instance
 type ReplicationAnalysis struct {
-	AnalyzedInstanceHostname                  string
-	AnalyzedInstancePort                      int
-	AnalyzedInstanceAlias                     string
-	AnalyzedInstancePrimaryAlias              string
-	TabletType                                topodatapb.TabletType
-	PrimaryTimeStamp                          time.Time
-	ClusterDetails                            ClusterInfo
-	AnalyzedInstanceDataCenter                string
-	AnalyzedInstanceRegion                    string
-	AnalyzedKeyspace                          string
-	AnalyzedShard                             string
+	AnalyzedInstanceHostname     string
+	AnalyzedInstancePort         int
+	AnalyzedInstanceAlias        string
+	AnalyzedInstancePrimaryAlias string
+	TabletType                   topodatapb.TabletType
+	PrimaryTimeStamp             time.Time
+	ClusterDetails               ClusterInfo
+	AnalyzedInstanceDataCenter   string
+	AnalyzedInstanceRegion       string
+	AnalyzedKeyspace             string
+	AnalyzedShard                string
+	// ShardPrimaryTimestamp is the primary term start time stored in the shard record.
+	ShardPrimaryTimestamp                     string
```

Comment:
Preferably this should be named `ShardPrimaryTermTimestamp`, and similarly the column name

Reply:
Done